README.md

```diff
@@ -15,8 +15,10 @@ Through this interface you should set the following configuration to the `Polari
 
 -   **loggerLevel** (_string_) - The level the logger is listening on, can be one of the following levels: `fatal` /
     `error` / `warn` / `info` / `trace` / `debug`.
--   **logstashConfigurations** (_LogstashConfiguration[] - optional_) - Through this property you can set multiple logstash
-    hosts and ports (**Notice that we use TCP to write logs to the logstash services**).
+-   **udpConfigurations** (_SocketAddress[] - optional_) - Through this property you can set multiple udp
```

Comment:
Instead of passing tcp/udp configurations, we should use the existing logstash config by adding a protocol option.